node/src/types/sync_leap.rs

```diff
@@ -191,10 +196,45 @@ impl Display for SyncLeap {
     }
 }
 
+#[derive(Clone, DataSize, Debug, Eq, PartialEq, Serialize)]
+pub(crate) struct SyncLeapValidationMetaData {
```

Comment:
can you put this into its own file, please?

Reply:
Certainly! Updated in 5ecdd81cdfaf9130fcaccac19e423214af429d3f